Sections/SmoothGear.cs

```diff
@@ -0,0 +1,54 @@
+using SimHub.Plugins;
+
+namespace DahlDesign.Plugin.Categories
+{
+    /// <summary>
+    /// Smooth Gear
+    /// </summary>
+    public class SmoothGear : SectionBase
```

Comment:
Tested and working, noticed error that would prevent a DDC wheel from getting an "N" in this prop when neutral is active on the wheel. Made a commit to fix it.